Personal Portfolio

ZohalMirkhel opened the pull request:
This pull request includes several updates and enhancements to the portfolio project to improve the overall user experience and design. The following changes have been implemented:

**CSS Enhancements:**

- Improved styling for the navigation bar, hero section, about me section, education section, and projects section.

- Added a smooth scroll behavior for a better navigation experience.

- Applied responsive design principles to mobile and PC.

- Enhanced the visual appeal with better color schemes, fonts, and layout adjustments.

ZohalMirkhel commented:
@walwalasafi, please take a look at my project, thank you! 
It does have linters erorr, but I will work on that later.

WALWALASAFI commented:
Dear @ZohalMirkhel, 
 i have checked your projects, 
1: it has linter errors, that u will fix it as soon as possible.
2: it doesnot have read me file.
3: it doesnot have demo link to have look to ur project.

ZohalMirkhel commented:
Dear @WALWALASAFI, please take a look at my project, I believe I am done with the desktop version of the website, except for the Show-Less and More button which still has some problems. Do let me know if there is anything else that I have to add or bring any changes.

WALWALASAFI commented:
@ZohalMirkhel 
your project still has a linters errors, beside of this your project has a clean  html and script code with great design.

ZohalMirkhel commented:
@WALWALASAFI, please take a look at my project. I believe I am done with making the website responsive. Look at all parts of the project and let me know if I have forgotten anything. However, I still have not worked on the show less button of the certificates page, linters error, and README.md file, kindly ignore those. Thank you!

ZohalMirkhel commented:
Dear @WALWALASAFI, did you check my project? Thank You!

ZohalMirkhel commented:
@ahzamir, Hello sir! Please take a look at my project. The UX/UI designers are yet to add pictures to the pop-up in mobile view and the footer, which they will be doing today. However, I submitted my project to avoid missing another deadline.

ZohalMirkhel commented:
Hi @ahzamir, could you please take a look at my project when you get a chance? I’d really appreciate your feedback. Thanks!

ahzamir commented:
@ZohalMirkhel 

Well, Done Zohal Mirkhel your project functionality is perfect, but in order to give feedback for the design and as a part of the requirements you need to add the Figma link on your README.md file and also add it to this comments replay.

ZohalMirkhel commented:
Sure sir @ahzamir, here is the link for the Figma design, https://www.figma.com/design/n2sJXMUcTX7VmMfxjVnrxQ/Untitled?node-id=0-1&t=XDZgs2yMGJzUZKsS-0 
Thank you!

ahzamir commented on style.css:
### Home Page:
> Overall impressive, perfect with the functionality just some minor Design issues:

1. The Header (Navbar):
  - Padding issue (Has more padding than the Figma)
  - Font Issue (Is not same as the Figma)
2.  Social Media (links icons):
  - Sizing (Should be bigger)
  - Placing (Should be centered on the page)
  - Implement for both Screen sizings
3. Contact Me Button:
  - Margin Top (Has less margin than the Figma)
  - Font Weight (Is not same as the Figma) - Implement for both screen sizings
 - This button should be a link to the Contact page (The contact page should be a separate page same as other pages, not a pop-up modal)

  - Your Design:
  
![image](https://github.com/user-attachments/assets/da6802a9-a64b-4388-af20-588726bce1cd)

  - Figma Design:
  
![image](https://github.com/user-attachments/assets/c13ecd7a-6db2-4087-9bd7-d9ac862e419c)

ahzamir commented on style.css:
### About Me Page:
> Overall impressive, perfect with the functionality just some minor Design issues:

1. Heading (About Me):
  - The background color (Is not the same as the Figma)
  - Implement for both screen sizings
2. The About Me Page text:
  - Line Height (Line spacing is not the same as the Figma design)
  - Implement for both screen sizing
  
  
  - Your Design:
  
![image](https://github.com/user-attachments/assets/4196c661-e9d6-4354-96be-baf3c1f71b02)

  - Figma Design:
  
![image](https://github.com/user-attachments/assets/321e72cd-755d-46f0-814d-2a49b43e1b1d)

ahzamir commented on style.css:
### Education Page:
> Overall impressive, perfect with the functionality just some minor Design issues:

  - Implement the same feedback for the `Heading`, `Text`, and `Button`:
  
  - Your Design:
  
![image](https://github.com/user-attachments/assets/024269f2-86a9-4630-a9a7-047d5d88b43a)

  - Figma Design:
  
![image](https://github.com/user-attachments/assets/ffbd83da-5280-46ca-9c62-47767df90cb2)

ahzamir commented on style.css:
### Projects Page:
> Overall impressive, perfect with the functionality just some minor Design issues, Impressive design functionality implemented for the see more project icon ⭐⭐⭐:

  - Implement the same feedback for the `Heading`, `Text`, and `Button` for both screen sizings.
  - Project's name are different from the Figma Design
  - Change your icon and make it exactly the same as the Figma design.
  - Padding issue (Mobile version).
  - Project's image, text and ... are smaller on figma design

  - Your Design:
  
![image](https://github.com/user-attachments/assets/c0add7d0-a8f2-4606-ba11-0e801e18a834)
![image](https://github.com/user-attachments/assets/c045d265-589c-47b2-a824-8abf9b7c9fb7)


  - Figma Design:
  
![image](https://github.com/user-attachments/assets/b8b8573b-e86d-4da2-b6a2-c0415432cd91)
![image](https://github.com/user-attachments/assets/d7281df4-8b77-4be7-bb3a-ccffffd8f220)

ahzamir commented on script.js:
### Certifications:
> Impressive design implemented, just implement the same feedback for the design as other pages:

- You need to add the certification links for every certification as it is part of the requirement:
  - You can add it on the image or certification name or separately.

ahzamir commented on style.css:
### Skills Page:
> Overall impressive, perfect with the functionality just some minor Design issues, Impressive design functionality implemented for the see more project icon ⭐⭐⭐:

  - Implement the same feedback for the `Heading` for both sizings.

ahzamir commented on style.css:
### Contact Page:
> This page should be a separated page or section the same as the other pages (Remove pop-up and add a separate page for the contact):

- Text Sizing (Is different from the Figma)
- Contact Page text padding (Is different from the Figma)
- The Footer background and text typography are not the same as the Figma Design.
- The mobile version is fine but has some little differences from the Figma

  - Your Design:
  
![image](https://github.com/user-attachments/assets/c6ed8f16-c840-40dd-89ee-df5aeb032446)

  - Figma Design:
    
![image](https://github.com/user-attachments/assets/944fe315-8bf6-488d-9d12-36acfd94b5bf)

ahzamir reviewed (changes requested):
### Hi @ZohalMirkhel 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Well done @ZohalMirkhel , Your hard work and passion served on this project resulted such an amazing project, but still your project has some minor issues you need to work on in order to get the final approval:
  - Check the comments below
  
# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

![AlmostThereReachGIF](https://github.com/LimaSahibzada/Survey-Form-project1/assets/96838030/4f0a3cd6-3a46-4e1e-8e31-ef617d4e8ac3)

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the JS Capstone Project and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
❌Your design is same as the Figma Designers Design

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

ahzamir commented on README.md:
```diff
@@ -1 +1,112 @@
+Note: No additional installations or dependencies are required for the JavaScript to work properly. The code runs directly in the browser.
+
+## Contribution
+1. **Hadisa Zahirzai**.
+2. **Zabeena Sherzoie**.
+3. **Safia Iqbal**.
```
- Well done, your README.md file is very great  just implement these 2 changes:
  - At least add one social profile link for the UI/UX Designers besides their name.
  - Add the link of the Figma Design on your README

ahzamir replied:
- Well done, your README.md file is very great  just implement these 2 changes:
  - At least add one social profile link for the UI/UX Designers besides their name.
  - Add the link of the Figma Design on your README

ZohalMirkhel commented:
Dear @ahzamir,
I believe I have completed the project to the best of my ability, incorporating all the feedback you provided. I also reviewed my work with my partner, and she confirmed that I had addressed all the feedback. I look forward to your response!

Thank You!

ahzamir commented on style.css:
- The button is linked to contact-popup but your contact page ID is #contact please solve this as your button does not have any function for now:

![image](https://github.com/user-attachments/assets/0890562c-63f1-4c00-9981-c8fe37b7cafd)

- Your contact form has lots of space on its top please solve this and instead add some margin bottom as your contact does not have any margin-bottom
- Your icons are too small compared to the Figma design please add some height and width to them
- When the user types in the input field it's white and hard to read, please make them black and give them some font-weight to make it look clear.

![image](https://github.com/user-attachments/assets/50e9c1b2-603e-41ea-86b3-a33e50151111)

ahzamir reviewed (approved):
### Hi @ZohalMirkhel 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.
- Congradulations Zohal, you have implemented all of the feedbacks and worked hard on this amazing project there is nothing to say other then merge your branch:
  - Before you merge your branch please have a look on 2 comments below and implement them.
- Well Done great design Implemented. 🎯🎉👍

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

![WinkThumbsUpGIF](https://github.com/ZohalMirkhel/palindrome/assets/96838030/e5ee293e-072a-4806-bde0-b64e13140efa)


 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the JS Capstone project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive
✔️Your Design is the same as the Figma design

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

ZohalMirkhel commented:
@ahzamir, thank you so much, sir. I brought the changes you suggested and then merged my branch. 
Thank you again!
